examples/tree_3d_dgsem/elixir_mhd_diffusive_alfven_wave.jl

```diff
@@ -0,0 +1,95 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the resistive compressible MHD equations
+
+prandtl_number() = 0.72
+mu() = 4e-2
+eta = 4e-2
+
+equations = IdealGlmMhdEquations3D(1.4)
+equations_parabolic = CompressibleMhdDiffusion3D(equations, mu = mu(),
+                                                 Prandtl = prandtl_number(),
+                                                 eta = eta,
+                                                 gradient_variables = GradientVariablesPrimitive())
+
+volume_flux = (flux_hindenlang_gassner, flux_nonconservative_powell)
+solver = DGSEM(polydeg = 3,
+               surface_flux = (flux_lax_friedrichs, flux_nonconservative_powell),
+               volume_integral = VolumeIntegralFluxDifferencing(volume_flux))
+
+coordinates_min = (-1.0, -1.0, -1.0) # minimum coordinates (min(x), min(y), min(z))
+coordinates_max = (1.0, 1.0, 1.0) # maximum coordinates (max(x), max(y), max(z))
+
+# Create a uniformly refined mesh with periodic boundaries
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 3,
+                n_cells_max = 50_000) # set maximum capacity of tree data structure
+
+function initial_condition_constant_alfven(x, t, equations)
```

Comment:
Is there a reference that could be added for this (simplified) Alfvén wave test case, e.g., the FLASH manual?